src/CreateEventQueryView/CreateEventQueryView.js

```diff
@@ -42,21 +42,26 @@ class CreateEventQueryView extends ConnectionComponent {
 	}
 	
 	submitQuery() {
+		let noValidationError = true;
```

Comment:
To me it would be easer to understand if this was "showValidationError", so the opposite of what you did.